app/views/keywords/edit.html.slim

```diff
@@ -0,0 +1,8 @@
+h1
+  | Editing keyword
+= render 'form', keyword: @keyword
+br
+div
+  = link_to 'Show this keyword', @keyword
+  |  |
```

Comment:
Is this an extra pipeline?

Reply:
Fixed up this file, it will be removed later unless it should be removed now? 🤔 @Lahphim